codecov has been discontinued

Github actions fails on installing dependencias because codecov has been discontinued as informed on https://github.com/codecov/codecov-python

Error message on action run:
ERROR: Could not find a version that satisfies the requirement codecov==2.1.12 (from versions: none)
ERROR: No matching distribution found for codecov==2.1.12
Error: Process completed with exit code 1.

Solution:
commenting/removing codecov from requirements.txt worked for me.

--- a/requirements.txt
+++ b/requirements.txt
@@ -18,7 +18,6 @@ black==22.3.0
 nose==1.3.7
 pinocchio==0.4.3
 coverage==6.3.2
-codecov==2.1.12
 
 # Utilities
 httpie==3.2.1

